pyveg/scripts/analyse_gee_data.py

```diff
@@ -284,15 +292,19 @@ def analyse_gee_data(input_location,
                 output_subdir = os.path.join(output_analysis_dir, "detrended")
                 run_time_series_analysis(ts_file, output_subdir, detrended=True)
 
-                ews_subdir = os.path.join(output_analysis_dir, "resiliance/deseasonalised")
-                run_early_warnings_resilience_analysis(ts_file, ews_subdir)
+                # resilience analysis only done in large enough time series
+                if size_ts > 12:
```

Comment:
since we have 12 a couple of times as this minimum size, we could maybe have it as a global parameter at the top of the file, like 
```
MIN_NUM_POINTS_FOR_EWS = 12
```